website/docs/configuration/functions/substr.html.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: "functions"
-page_title: "substr function"
+page_title: "substr - Functions - Configuration Language"
```

Comment:
In retrospect, I really should have done these function page titles in a separate PR to cut down on the noise here. 😑